src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/DefaultRazorSemanticTokenInfoServiceTest.cs

```diff
@@ -19,14 +20,55 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Test.Semantic
 {
     public class DefaultRazorSemanticTokenInfoServiceTest : DefaultTagHelperServiceTestBase
```

Comment:
Eventually this would benefit greatly from snapshot testing.

Reply:
Oh, like https://github.com/dotnet/aspnetcore-tooling/blob/2749fac6eccd59d3223649e8c6f6cff88daec4e6/src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/README.md but for Semantic Tokens? That's a good idea, I filed https://github.com/dotnet/aspnetcore/issues/26186.